LiskHQ/lisk-desktop
# Remove testnet from default network

### What was the problem?

This PR resolves #5536

### How was it solved?

:recycle: Update default networks

### How was it tested?

- Check network drop and by default you should see Mainnet
- Add testnet and other network
